src/tools/storage-perf/CMakeLists.txt

```diff
@@ -2,20 +2,20 @@
 #
 # This source code is licensed under Apache 2.0 License.
 
-nebula_add_executable(
-    NAME
-        storage_perf
-    SOURCES
-        StoragePerfTool.cpp
-    OBJECTS
-        ${tools_test_deps}
-    LIBRARIES
-        ${ROCKSDB_LIBRARIES}
-        ${THRIFT_LIBRARIES}
-        ${PROXYGEN_LIBRARIES}
-        gtest
-        curl
-)
+#nebula_add_executable(
+#    NAME
+#        storage_perf
```

Comment:
why comment out this tools (simple_kv_verify, storage_perf, storage_integrity), is it related that toss is turned off?

Reply:
Not related to toss, these tools are out of date now, which has not been maintained for a long time.